engine/order_manager_test.go

```diff
@@ -645,6 +638,38 @@ func TestSubmit(t *testing.T) {
 	}
 }
 
+// TestSubmitOrderAlreadyInStore ensures that if an order is submitted, but the WS sees the conf before processSubmittedOrder
+// then we don't error that it was there already, and we merge together the details
```

Comment:
You're not really merging together details, like I said above, you're discarding them and this test can demonstrate that (by not reusing the same struct which wouldn't occur with a REST order and WS update). Please update comment

Reply:
Agreed and Done: 69043784

I've simply removed the tail of this sentence.
I haven't gone so far as to add a failure test to *prove* we lose the detalis, since that's not a feature we want to protect against regressing.
If someone fixes that, AND the ClientOrderID is still correct (which _is_ tested) then we're happy.